content/cloud/github-actions.md

```diff
@@ -0,0 +1,190 @@
+
+Now let's add a Spin application using a template that includes Spin's HTTP request handler. Choose your preferred language.
+
+<!-- @selectiveCpy -->
+
+```bash
+# Create new Spin application
+$ spin new
+Pick a template to start your application with:
+  http-c (HTTP request handler using C and the Zig toolchain)
+  http-csharp (HTTP request handler using C# (EXPERIMENTAL))
+  http-empty (HTTP application with no components)
+  http-go (HTTP request handler using (Tiny)Go)
+  http-grain (HTTP request handler using Grain)
+  http-js (HTTP request handler using Javascript)
+  http-php (HTTP request handler using PHP)
+> http-py (HTTP request handler using Python)
+  http-rust (HTTP request handler using Rust)
+  http-swift (HTTP request handler using SwiftWasm)
+  http-ts (HTTP request handler using Typescript)
+  http-zig (HTTP request handler using Zig)
+  redirect (Redirects a HTTP route)
+  redis-go (Redis message handler using (Tiny)Go)
+  redis-rust (Redis message handler using Rust)
+  static-fileserver (Serves static files from an asset directory)
+```
+
+For this tutorial, we've decided to use Python (`http-py`). Please feel free to choose your preferred supported Spin language (when prompted to select an HTTP request handler template).
+
+Once you have selected your preferred template, follow the prompts to complete the creation of your new application i.e. name the application `spin-python-app`, then add a description (and just hit enter to accept the default HTTP base and HTTP path options):
+
+<!-- @nocpy -->
+
+```console
+Enter a name for your new application: spin-python-app
+Description: A python application to test GitHub Actions.
+HTTP base: /
+HTTP path: /...
+```
+
+### Setting Up Github Actions
+
+The first step, in setting up GitHub Actions, is to create a new folder called `workflows` (inside a new parent folder called `.github`). We use this new folder structure to keep a GitHub Action `.yaml` file (that we will create in a minute). For now, please go ahead and create the following directories in the base of your repository dir. For example:
+
+<!-- @selectiveCpy -->
+
+```bash
+# Change into ~/github-actions-tutorial dir
+$ cd ~/github-actions-tutorial
+# Create workflows directory (and make parent directory as necessary via the -p option)
+$ mkdir -p .github/workflows/
+```
+
+Now it's time to add the `fermyon/actions/spin/deploy` GitHub Action. To do so, please create a file called `spin-deploy-action.yaml`:
+
+<!-- @selectiveCpy -->
+
+```bash
+$ cd ~/github-actions-tutorial/.github/workflows/
+$ vi spin-deploy-action.yaml
+```
+
+Then add the following code, to the `spin-deploy-action.yaml` file:
+
+<!-- @nocpy -->
+
+```yml
+name: spin-deploy
+
+on:
+  - push
+
+jobs:
+  spin:
+    runs-on: ubuntu-latest
+    name: Build and deploy
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup `spin`
+        uses: fermyon/actions/spin/setup@v1
+        with:
+          version: canary
```

Comment:
should we use the default version instead of `canary` here (if we remove `version` field, it installs latest by default)

Reply:
Yes, good call (I removed the version field altogether)